src/ruby/spec/pb/codegen/package_option_spec.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require 'open3'
+require 'tmpdir'
+
+describe 'Code Generation Options' do
+  it 'should generate and respect package options' do
+    unless system('which grpc_ruby_plugin', out: File::NULL) && system('which protoc', out: File::NULL)
```

Comment:
whoops, actually, can we please change the way this test find the `grpc_ruby_plugin` and `protoc` so that is always runs and won't ever get skipped during c-builds?

See this test for an example of doing that: https://github.com/grpc/grpc/blob/master/src/ruby/end2end/package_with_underscore_checker.rb#L19 (the way that the test finds `protoc` and `grpc_ruby_plugin` executables can be copied over this test as well.

Reply:
haha - no worries happy to make the change (wasn't sure how invasive you'd want this)

Reply:
done